web/src/components/PureMarkdownContent.tsx

```diff
@@ -49,6 +49,13 @@ function customRemarkPlugin():
             header: node.attributes.header,
           };
         }
+        if (node.name === "note") {
+          const data = node.data || (node.data = {});
+          data.hName = "note";
+          data.hProperties = {
+            header: node.attributes.header,
+          };
+        }
```

Comment:
[dust] This section might be a good candidate for a `switch`.

Reply:
[pebble] A lot of this (lines 38-66) is redundant. We could rewrite this to be a little more efficient and possibly easier to read.